src/compile/selection/selection.ts

```diff
@@ -137,9 +137,12 @@ export function assembleUnitSelectionSignals(model: UnitModel, signals: any[]) {
     });
   });
 
-  const facetModel = getFacetModel(model);
-  if (signals.length && facetModel) {
-    const name = stringValue(facetModel.getName('cell'));
+  return signals;
+}
+
+export function assembleFacetSignals(model: FacetModel, signals: any[]) {
```

Comment:
Not necessary for this PR, but now that we have typings for Vega signals, we should try to replace the `any`s. https://github.com/vega/vega-typings/blob/master/types/spec/signal.d.ts

Reply:
Agreed! I've replaced a bunch of them in my refactoring PR (#4068, there may be more though). I branched this one off master to keep it unblocked.